demo/util/confirm-features.mjs

```diff
@@ -0,0 +1,25 @@
+function redDiv(textContent) {
+  const div = document.createElement('div');
+  div.style.color = 'red';
+  div.textContent = textContent;
+  return div;
+}
+
+/**
+ * Checks that the features needed are present in the browser. If not,
+ * it places error messages inside |element|.
+
+ **/
+export function confirm(element) {
+  if (!element['displayLock']) {
```

Comment:
Nit: `element.displayLock` would be a bit more usual.

Reply:
For some reason I thought it would be an error to access it if it doesn't exist. Fixed.